spec/acceptance/exec_powershell_spec.rb

```diff
@@ -34,6 +34,31 @@
 
   end
 
+  describe 'should handle a try/catch successfully' do
```

Comment:
Yeah, and I'm not sure that we're actually verifying that here either @ferventcoder, given the assertion is:

```ruby
apply_manifest(p1, :expect_changes => true, :future_parser => FUTURE_PARSER)
```

Beaker docs say:

```
        # @option opts [Boolean]  :expect_changes (false) This option enables
        #                         detailed exit codes and causes a test failure
        #                         if `puppet --apply` indicates that there were
        #                         no resource changes during its execution.
```

I don't see mention in the original [MODULES-2634](https://tickets.puppetlabs.com/browse/MODULES-2634) ticket about the Puppet exit status or whether or not it thought it applied resources, in the case where the the PS code did not run... 


So I'm not sure the test as written here even does anything to verify whether or not the PS code executed - another reason I'd prefer to move to something very explicit that we can verify on the system (i.e. writing a file like the rewritten test). Then we can clear up any doubts...